src/neo/IO/Helper.cs

```diff
@@ -135,12 +192,23 @@ public static int GetVarSize<T>(this IReadOnlyCollection<T> value)
             return GetVarSize(value.Count) + value_size;
         }
 
+        /// <summary>
+        /// Gets the size of the specified <see cref="string"/>.
```

Comment:
Mention relation to serialization format? Otherwise the size of string can be discovered in an easier fashion.

Reply:
Updated.